Source/.clang-format

```diff
@@ -66,8 +66,8 @@ PenaltyBreakString: 1000
 PenaltyExcessCharacter: 1000000
 PenaltyReturnTypeOnItsOwnLine: 60
 PointerAlignment: Left
-ReflowComments:  true
-SortIncludes:    true
+ReflowComments:  Always
```

Comment:
We're still on `clang-format-13` where this still a false/true value.
Waiting on #13436 for clang-format-19.

Reply:
Despite the documentation, clang-format-19 doesn't seem to like `ReflowComments: Always`.